packages/block-library/src/block/test/edit.native.js

```diff
@@ -72,25 +138,19 @@ describe( 'Reusable block', () => {
 			getByA11yLabel( /Reusable block Block\. Row 1/ )
 		);
 
-		const reusableBlockRequest = fetchRequest.mock.calls.find( ( call ) =>
-			call[ 0 ].path.startsWith( endpoint )
-		);
-
 		const blockDeleted = await waitFor( () =>
 			within( reusableBlock ).getByText(
 				'Block has been deleted or is unavailable.'
 			)
 		);
 
-		expect( reusableBlockRequest ).toBeDefined();
```

Comment:
I decided to remove this check because it's already covered when verifying the existence of the reusable block.